three.js/src/threex/arjs-markercontrols.js

```diff
@@ -269,6 +266,20 @@ MarkerControls.prototype._initArtoolkit = function(){
 		var modelViewMatrix = new THREE.Matrix4().fromArray(event.data.matrix)
 		_this.updateWithModelViewMatrix(modelViewMatrix)
 	}
+
+	function onGetMarker(event){
```

Comment:
The function body could use some formatting but I wanted to keep the PR lean and just copied the code from the inline event handler...